pkgs/tools/misc/z/default.nix

```diff
@@ -0,0 +1,45 @@
+{ fetchFromGitHub, lib, stdenv }:
+  stdenv.mkDerivation
+    { pname = "z";
+      version = "1622481819";
+      # ^ there isn't a version, so the UNIX time of the commit is used
+
+      src =
+        fetchFromGitHub
+          { owner = "rupa";
+            repo = "z";
+            rev = "b82ac78a2d4457d2ca09973332638f123f065fd1";
+            sha256 = "0phk9lswwxsypchb11qsnxy1pv5xg1zkrqj8im6x2icma63hfcz2";
+          };
+
+      phases = [ "unpackPhase" "installPhase" ];
+
```

Comment:
```suggestion
```
This is forbidden for leaf packages.

Reply:
What am I supposed to do instead, disable all the phases I don't want with the relevant attributes?